test/verify/check-machines

```diff
@@ -126,6 +126,14 @@ VOLUME_XML="""
 class TestMachines(MachineCase):
     created_pool = False
 
+    def setUp(self):
+        MachineCase.setUp(self)
+
+        # enforce use of cockpit-machines instead of cockpit-ovirt
```

Comment:
Do RPMs of the two normally conflict for real users? Or is it the case that normally when cockpit-ovirt is installed it overrides cockpit-machines. If so this should be documented in doc  features page.

Reply:
Packages are not mutually exclusive but cockpit-ovirt overrides cockpit-machines (manifest.json:priority). Documentation is updated.